### PR TITLE
Distributed Data: General Typed docs cleanup after all API changes

### PR DESCRIPTION
**Pull request by helena**
#27764 Initial push is rough, analysis paralysis during travel, comparing on just laptop screen. Easier to see in diff.

### CODE REVIEW

File: akka-docs/src/main/paradox/distributed-data.md [hunk 1]
```diff
@@ -358,241 +245,21 @@ types that support both updates and removals, for example `ORMap` or `ORSet`.
...
-Java
-: @@snip [DistributedDataDocTest.java](/akka-docs/src/test/java/jdocs/ddata/DistributedDataDocTest.java) { #lwwregister }
-
-Instead of using timestamps based on `System.currentTimeMillis()` time it is possible to
-use a timestamp value based on something else, for example an increasing version number
-from a database record that is used for optimistic concurrency control.
-
-Scala
-: @@snip [DistributedDataDocSpec.scala](/akka-docs/src/test/scala/docs/ddata/DistributedDataDocSpec.scala) { #lwwregister-custom-clock }
+### delta-CRDT
```
Comment (johanandren) [inline akka-docs/src/main/paradox/distributed-data.md]: Shouldn't it start with an uppercase D though?

**Review by johanandren — APPROVED** [review]
LGTM

File: akka-docs/src/main/paradox/distributed-data.md [hunk 2]
```diff
@@ -204,58 +169,11 @@ Scala
...
-For small clusters (<7) the risk of membership changes between a WriteMajority and ReadMajority
-is rather high and then the nice properties of combining majority write and reads are not
-guaranteed. Therefore the `ReadMajority` and `WriteMajority` have a `minCap` parameter that
-is useful to specify to achieve better safety for small clusters. It means that if the cluster
-size is smaller than the majority size it will use the `minCap` number of nodes but at most
-the total size of the cluster.
+See @ref:[Distributed Data - Get](typed/distributed-data.md#get) in the documentation of the new APIs. 
 
+### Consistency
+
```
Comment (patriknw) [inline akka-docs/src/main/paradox/distributed-data.md]: link to typed/distributed-data.md#consistency description?

File: akka-docs/src/main/paradox/distributed-data.md [hunk 3]
```diff
@@ -69,6 +52,24 @@ Scala
 Java
 : @@snip [DataBot.java](/akka-docs/src/test/java/jdocs/ddata/DataBot.java) { #data-bot }
 
+### Subscribe
```
Comment (patriknw) [inline akka-docs/src/main/paradox/distributed-data.md]: better to have this after "Get"

File: akka-docs/src/main/paradox/typed/distributed-data.md [hunk 4]
```diff
@@ -59,16 +65,26 @@ Although you can interact with the `Replicator` using the @scala[`ActorRef[Repli
 from @scala[`DistributedData(ctx.system).replicator`]@java[`DistributedData(ctx.getSystem()).replicator()`] it's
 often more convenient to use the `ReplicatorMessageAdapter` as in the above example.
 
+### Subscribe
```
Comment (patriknw) [inline akka-docs/src/main/paradox/typed/distributed-data.md]: better to have this after "Get"

File: akka-docs/src/main/paradox/distributed-data.md [hunk 5]
```diff
@@ -22,24 +22,7 @@ to see what this looks like in practice.
 
 ## Introduction
```
Comment (patriknw) [inline akka-docs/src/main/paradox/distributed-data.md]: above there is a link:
```
@ref:[distributed-data](typed/distributed-data.md)
```
that should be 
```
@ref:[Distributed Data](typed/distributed-data.md)
```

but maybe that is already changed elsewhere?

**Review by patriknw — COMMENTED** [review]
looking good, just a few small things

File: akka-docs/src/main/paradox/distributed-data.md (hunk 5 above)
Comment (helena) [reply akka-docs/src/main/paradox/distributed-data.md]: yes, though I learned if you use the label name, paradox adds the text for you. But I like the explicit better :)

**Comment by helena** [comment]
Updated

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**